examples/riscv_mini/setup.py

```diff
@@ -16,9 +16,7 @@
         "riscv_mini",
     ],
     install_requires=[
-        "magma-lang>=2.1.19",
-        "mantle",
-        "mantle2",
+        "magma-lang",
```

Comment:
should this still have a version minimum? or does that screw with the re-install?